rosette/h/Heap.h

```diff
@@ -197,21 +188,19 @@ class ProtectedItem {
     friend void Init_Heap(EMPTY);
 
    public:
-    ProtectedItem(void*);
-    ~ProtectedItem(EMPTY);
-};
+    ProtectedItem(void* v) : next(root), item(v) {
+        root = this;
+    }
 
-inline ProtectedItem::ProtectedItem(void* v) : next(root), item(v) {
-    root = this;
-}
+    ~ProtectedItem() { root = next; }
+};
 
-inline ProtectedItem::~ProtectedItem(EMPTY) { root = next; }
 
 
 #define PROTECT(v) ProtectedItem name2(_, v)(&v)
 #define PROTECT_THIS(type) \
     type* __this__ = this; \
-    PROTECT(__this__)
+PROTECT(__this__)
```

Comment:
Extraneous whitespace change, the original was better.

Reply:
Done.